app/helpers/application_helper.rb

```diff
@@ -51,4 +51,8 @@ def search_form_class
   def active?(path)
     "is-active" if request.path_info == path
   end
+
+  def without_count_paginate(items)
```

Comment:
I think `paginate_without_count` looks better. But both describes actual implementation detail and not the expected output of this helper (having pagination missing numbered steps -> prev/next links only). What about some more generic name? Maybe `simple_paginate` (and leave a comment about performance in here)?